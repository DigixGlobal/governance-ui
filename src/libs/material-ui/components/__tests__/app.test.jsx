import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '../app';

jest.mock('react-redux', () => ({
  connect: () => component => component,
}));

describe('<App />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <App ready />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
