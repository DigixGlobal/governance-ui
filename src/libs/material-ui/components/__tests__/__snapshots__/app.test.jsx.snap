// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="pusher"
>
  <TransactionSigningOverlay />
  <HashRouter>
    <MenuSystem
      className="content"
      dropdown={false}
      hidden={false}
      marginTop="5em"
      menuProps={
        Object {
          "borderless": true,
          "fixed": "top",
        }
      }
      parentRoute="/"
      renderLastItem={[Function]}
      tabs={
        Array [
          Object {
            "component": [Function],
            "icon": "key",
            "name": "Keystores",
            "path": "/keystores",
          },
          Object {
            "component": [Function],
            "icon": "wrench",
            "name": "Config",
            "path": "/config",
          },
        ]
      }
      usingRouter={true}
    />
  </HashRouter>
  <Footer />
</div>
`;
